src/Workspaces/Core/Portable/Remote/RemoteHostClient.cs

```diff
@@ -97,6 +103,7 @@ public static string CreateClientId(string prefix)
         /// <summary>
         /// Creates <see cref="SessionWithSolution"/> for the <paramref name="serviceName"/> if possible, otherwise returns <see langword="null"/>.
         /// </summary>
+        [Obsolete("Used by LUT", error: false)]
```

Comment:
💡 If this is only used by LUT, can we move it into the external access code for LUT?